carla_integration/carma.env

```diff
@@ -5,10 +5,10 @@
 #  Logging
 ###
 export ROSCONSOLE_CONFIG_FILE="/opt/carma/vehicle/config/carma_rosconsole.conf"
-export ROSCONSOLE_FORMAT='${time} | ${severity} | ${function}:${line} | ${message}'
-#log_name=$(date +%Y%m%d-%H%M%S) # TODO unfortunately this doesn't work as this file is run independantly by each container
```

Comment:
Without this do we not have a datestamped folder for each log file?

Reply:
Correct. We have not had this functionality since moving to Docker in CARMA 2. All I'm doing here is removing the comment line since its been 2 years since its been of relevance.